juniper/src/macros/tests/args.rs

```diff
@@ -73,13 +73,12 @@ impl Root {
         0
     }
 
-    // TODO: enable once [RFC 2565](https://github.com/rust-lang/rust/issues/60406) is implemented
-    // fn attr_arg_descr(#[doc = "The arg"] arg: i32) -> i32 { 0 }
-    // fn attr_arg_descr_collapse(
-    //     #[doc = "The arg"]
-    //     #[doc = "and more details"]
-    //     arg: i32,
-    // ) -> i32 { 0 }
+    fn attr_arg_descr(arg: i32) -> i32 {
```

Comment:
This looks wrong? Also, we have https://github.com/graphql-rust/juniper/pull/441 outstanding

Reply:
Got it. 
I put it back to todos and updated a link to #441